app/upw/controllers/saveAndContinue.js

```diff
@@ -72,8 +55,7 @@ class SaveAndContinue extends BaseSaveAndContinue {
 
   saveValues(req, res, next) {
     const answers = req.sessionModel.get('formAnswers') || {}
-    const answersWithInvalidatedDeclarations = invalidateDeclarations(answers)
-    req.sessionModel.set('answers', answersWithInvalidatedDeclarations)
+    req.sessionModel.set('answers', answers)
```

Comment:
Just need to double check that this is still the right thing to do, after the changes we made yesterday.

Reply:
Good point! will rebase against `main` and double check

Reply:
Rebased and this is looking okay in the diff